<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_LineBisectorOptions</class>
  <widget class="QWidget" name="QG_LineBisectorOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>280</width>
        <height>22</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>0</hsizetype>
        <vsizetype>0</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>220</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>280</width>
        <height>22</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Line Bisector Options</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QLabel" name="lLength">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Length:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leLength">
          <property name="toolTip" stdset="0">
            <string>Length of bisector</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lNumber">
          <property name="text">
            <string>Number:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QSpinBox" name="sbNumber">
          <property name="minimum">
            <number>1</number>
          </property>
          <property name="toolTip" stdset="0">
            <string>Number of bisectors to create</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <connections>
    <connection>
      <sender>leLength</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_LineBisectorOptions</receiver>
      <slot>updateLength(QString)</slot>
    </connection>
    <connection>
      <sender>sbNumber</sender>
      <signal>valueChanged(int)</signal>
      <receiver>QG_LineBisectorOptions</receiver>
      <slot>updateNumber(int)</slot>
    </connection>
  </connections>
</ui>
